docs/src/snap/howto/cos-lite.md

```diff
@@ -0,0 +1,142 @@
+# Integrating with COS Lite
+
+**Canonical Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between Canonical Kubernetes and COS Lite (Canonical
+Observability Stack). This guide will help you integrate a COS Lite
+deployment with a Canonical Kubernetes deployment.
+
+This document assumes you have a controller with an installation of
+Canonical Kubernetes. If this is not your case, refer to
+["Installing Canonical Kubernetes"][how-to-install].
+
+## Preparing a platform for COS Lite
+
+First, create a Microk8s model to act as a deployment cloud for COS Lite:
+
+```
+juju add-model --config logging-config='<root>=DEBUG' microk8s-ubuntu aws
+```
+
+We also set the logging level to DEBUG so that helpful debug information is
+shown when you use `juju debug-log` (see [juju debug-log][juju-debug-log]).
+
+Note: If you are unfamiliar with Juju models, the documentation can be found [here][juju-models].
```

Comment:
```suggestion
> Note: If you are unfamiliar with Juju models, the documentation can be found [here][juju-models].
```

Reply:
I would relocate this to underneath the heading at Line 17. It probably doesn't need a formal note, but a line explaining  that we are going to add a new model (see Juju docs for more on models) to keep the observability separate from the K8s model